Update IAM policy settings for service accounts

anikundesu opened the pull request:
Update IAM policy settings for $CitrixServiceAccount and Cloud Build service account to follow the document from Citrix.
https://docs.citrix.com/en-us/citrix-virtual-apps-desktops-service/install-configure/resource-location/google.html

anikundesu committed:
Update IAM policy settings for service accounts

Update IAM policy settings for $CitrixServiceAccount and Cloud Build service account to follow the document from Citrix.
https://docs.citrix.com/en-us/citrix-virtual-apps-desktops-service/install-configure/resource-location/google.html

jeffallen-google commented on deploy.ps1:
```diff
@@ -163,6 +163,13 @@ $ServiceAccount = $AdminServiceAccount
 
 Write-Host "Service Account: [$ServiceAccount]"
 
+$CloudbuildServiceAccount = "$ProjectNumber@cloudbuild.gserviceaccount.com"
```
Referencing the Cloud Build service account here assumes that API has been enabled for the project. As such "cloudbuild" should be added to the list of required APIs on line 85.

anikundesu replied:
I added cloudbuild as one of required APIs on line 85. This requirement is also written in Citrix's document.

jeffallen-google reviewed (changes requested):
Please include cloudbuild in required APIs to ensure the API is enabled and service account exists.

anikundesu committed:
Update deploy.ps1

Add "cloudbuild" as one of required APIs. Cloud Build API is required at the [Citrix document](https://docs.citrix.com/en-us/citrix-virtual-apps-desktops-service/install-configure/resource-location/google.html) .
